src/d3a/models/strategy/storage.py

```diff
@@ -152,9 +146,29 @@ def _set_be_alternative_pricing(self):
             else:
                 raise MarketException
 
+    @staticmethod
+    def _validate_break_even_points(break_even):
+        if type(break_even) not in [tuple, list, dict]:
+            raise ValueError("Break even points have to be a tuple, list or dict.")
+
+        if type(break_even) is list:
```

Comment:
Can you please explain the reason for the whole if/elif block here?
AFAIK `read_arbitrary_profile` can only handle dicts, file-name stringe or json strings.
Did you try it with tupple?
Additionally, in the general PR comment you stated `Throughout the process I refactored the BE to always use a namedtuple`. However, in this `if type(break_even) is list:` block, it is not converted into a BreakEven instance.

Reply:
Point #1: Apparently the read_arbitrary_profile can also handle tuples. See https://github.com/gridsingularity/d3a/blob/master/src/d3a/models/read_user_profile.py#L239. Break even always used to be a tuple, and it always converted the values correctly as far as I have seen. 
Point #2: If you pass a list in this function, it will be converted to a tuple (it is the next line in the diff) and will be treated as a tuple from then on, therefore it will correctly be converted to a namedtuple instance.

Reply:
Thanks for explaining! Makes sense! I was a little confused here because i though the user can upload a tuple of tuple (a profile of Breakeven tuples).
And I have to read more carefully in the future and dont confuse a elif with an if :-)